src/Caliburn.Micro.Maui/Caliburn.Micro.Maui.csproj

```diff
@@ -0,0 +1,98 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<TargetFrameworks>net6.0;net6.0-android;net6.0-ios;net6.0-maccatalyst</TargetFrameworks>
+		<TargetFrameworks Condition="$([MSBuild]::IsOSPlatform('windows')) and '$(MSBuildRuntimeType)' == 'Full'">$(TargetFrameworks);net6.0-windows10.0.19041</TargetFrameworks>
+    <TargetsForTfmSpecificBuildOutput>$(TargetsForTfmSpecificBuildOutput);CopyProjectReferencesToPackage</TargetsForTfmSpecificBuildOutput>
```

Comment:
Do we need to add RuntimeIdentifiers?  I was getting an error when working on winui about no support for win10-arm64

Reply:
The RuntimeIdentifiers are for the platform specifics because we still hook in rather early. I was pondering doing away with the platform specifics but I think it is nice to have the early option to register platform services for IoC.

Reply:
We also might have some overlap since MAUI uses WinUI. Maybe there is code to be shared?